### PR TITLE
SPORTSHUB-230 Organiser mobile dashboard + Organiser navbar

### PR DESCRIPTION
**Pull request by Edwin-Ni**
## Why was your change needed?

Added mobile view to organiser dashboard and a mobile navbar for organiser

### CODE REVIEW

**Comment by owya490** [comment]
where are the nav bar changes?

File: frontend/app/(no-footer)/organiser/dashboard/page.tsx [hunk 1]
```diff
@@ -71,13 +71,11 @@ export default function Dashboard() {
...
-                <h1 className="text-2xl font-bold text-center w-full sm:w-[300px] xl:w-[290px] 2xl:w-[320px]">
-                  Upcoming Events
-                </h1>
+            <div className="md:overflow-auto mt-6 lg:mt-0">
+              <div className="bg-organiser-light-gray py-4 rounded-2xl lg:px-8">
+                <h1 className="text-2xl font-bold text-center w-full">Upcoming Events</h1>
               </div>
-              <div>
+              <div className="sm:grid sm:grid-cols-2 sm:gap-2 lg:block">
                 {eventDataList
```
Comment (owya490) [inline frontend/app/(no-footer)/organiser/dashboard/page.tsx:79]: the eventDataList get chopped off at the bottom in mobile view:

**Review by 41dan — COMMENTED** [review]
lgtm

**Review by 41dan — APPROVED** [review]
lgtm

**Review by owya490 — APPROVED** [review]
thank you!

**Pull request merged**